docs/api.rst

```diff
@@ -55,6 +55,24 @@ AutoShardedClient
 
 .. autoclass:: AutoShardedClient
     :members:
+    
+Application_Command    
```

Comment:
underscores for a title?

Reply:
@ooliver1 i did it cause all of these are stored in application_command.py

Reply:
Title: Application Commands

Reply:
@VincentRPS i cant unresolve stuff but this hasnt been changed

Reply:
@RPSMain

Reply:
i can't commit it waa